audio/src/backend/openal.cpp

```diff
@@ -644,7 +644,7 @@ void OpenAL::doInput()
 
     applyGain(inputBuffer, AUDIO_FRAME_SAMPLE_COUNT_TOTAL, gainFactor);
 
-    float volume = getVolume();
+    auto volume = static_cast<qreal>(getVolume());
```

Comment:
I'm surprised this change doesn't cause a warning on line 657 where `IAudioSource::volumeAvailable(float)` is called. :x Looks like there's a lot of back and forth between double and float in the audio code.

Reply:
Removed some flipping back and forth of types, OpenAl::getVolume now returns qreal and IAudioSource::volumeAvailable now sends qreal.

That removes this case.